### ISSUE TITLE
Cotrollers --> Checkbox (List selected)

### ISSUE THREAD

**Issue by lmslachciak** [Samsung/TAU#722]
Controllers --> Checkbox (List selected)
verify styles, list selected

**Comment by TomaszLukawskiSam** [comment]
@lslachciak 
After analyzing this issue, I came to the conclusion that it is rather a task to create a mutliple selection list.

**Comment by lslachciak** [comment]
Yes this is blocked by multiple selection for list

**Comment by TomaszLukawskiSam** [comment]
I'll reopen PR referring to this issue.

**Comment by TomaszLukawskiSam** [comment]
@lslachciak 
This issue partly overlaps with the scope of the lists (multiple selection) and in my opinion should be parted to several smaller.
    - changes in AppBar for add checkbox "All" in header, (#847)
    - changes in AppBar for posibility to modify AppBar title regarding to number of selected items
    - showing the BottomBar if occures selected items
    - text within a list item should be considered part of the primary action target (in this case: taping on `li` area is changing checkbox state)

**Comment by lslachciak** [comment]
First two item I agree.
Regarding  
_- showing the BottomBar if occures selected items_
I see on guide screens that regardless items are selected or not bottom bar is visible with "move/share/delete" options.

_ -text within a list item should be considered part of the primary action target_ 
could you explain more?

**Comment by TomaszLukawskiSam** [comment]
> _ -text within a list item should be considered part of the primary action target_
> could you explain more?

This is requirement from guideline and I understand this that taping on `li` area is changing checkbox state

**Comment by lslachciak** [comment]
ok, now all requirements are clear
but 
_ -text within a list item should be considered part of the primary action target_
is for list requirement, out of this item scope

**Comment by lslachciak** [comment]
so we have here:
~~https://github.com/Samsung/TAU/issues/847~~
https://github.com/Samsung/TAU/issues/851
https://github.com/Samsung/TAU/issues/852

**Comment by lslachciak** [comment]
also please note that we need example under Controllers->Checkbox (List Selected)

**Comment by TomaszLukawskiSam** [comment]
Do you want to duplicate sample file or add only entry in index.html with reference to sample file in list examples?

**Comment by TomaszLukawskiSam** [comment]
I added that link to PR: https://github.com/Samsung/TAU/pull/864

**Comment by TomaszLukawskiSam** [comment]
I think that all task related to this issue is closed. We can close the issue.

**Comment by lslachciak** [comment]
@TomaszLukawskiSam the "Cancel" button in appbar shouldn't deselect elements?
Would it be ease to handle this?
![image](https://user-images.githubusercontent.com/2246331/78528045-c03eaf00-77de-11ea-911e-4d7234effe2f.png)

UIComponents app will be checked by testers and I afraid that this will be a potential issue.

**Comment by TomaszLukawskiSam** [comment]
@lslachciak 
I understand your point of view but,
in my opinion "cancel" shouldn't deselect elements but should hide "select all" and checkboxes, so this is rather feature of listview like change mode of listview.

I think this is reason to create new issue.

**Comment by lslachciak** [comment]
ok
